webassets_elm/__init__.py

```diff
@@ -30,16 +44,19 @@ def input(self, _in, out, **kw):
         the compiled contents to a temporary file and then read it in order to
         output to stdout.
         """
-
         # create a temp file
         tmp = NamedTemporaryFile(suffix='.js', delete=False)
         tmp.close()  # close it so windows can read it
 
         # write to a temp file
         elm_make = self.binary or 'elm-make'
+        change_directory = bool(self.change_directory or False)
         source = kw['source_path']
+        source_dir = os.path.join(*os.path.split(source)[:-1])
```

Comment:
IMHO this is not as readable as it could be. What about `source_dir = os.path.dirname(source)`?